webknossos-datastore/app/com/scalableminds/webknossos/datastore/datareaders/zarr3/Codecs.scala

```diff
@@ -294,8 +287,16 @@ final case class ShardingCodecConfiguration(chunk_shape: Array[Int],
                                             index_codecs: Seq[CodecConfiguration],
                                             index_location: IndexLocationSetting.IndexLocationSetting =
                                               IndexLocationSetting.end)
-    extends CodecConfiguration {
+    extends CodecConfiguration with BoxImplicits {
   override def name: String = ShardingCodecConfiguration.name
+// index_codecs.size <= 2 && index_codecs.count(_.name == "bytes") == 1 && index_codecs.count(_.name == "crc32c") <= 1
+  def isSupported: Box[Unit] =
+  for {
+    _ <- bool2Box(index_codecs.size <= 2) ?~! "Maximum of 2 index codecs supported"
+    _ <- bool2Box(index_codecs.count(_.name == "bytes") == 1) ?~! "Exactly one bytes codec supported"
```

Comment:
can we get something like "Only exactly one bytes codec is supported, got 3"? (for all of these?)